test/sample_files/no_struct_spec_module.ex

```diff
@@ -1,3 +1,5 @@
-defmodule Doctor.NoStructSpecModule do
+defmodule Doctor.OpaqueStructSpecModule do
```

Comment:
The file names for the two new test modules need to be flipped. The `OpaqueStructSpecModule` module has a filename of `no_struct_spec_module.ex`